Documentation/NuttxPortingGuide.html

```diff
@@ -2824,100 +2824,13 @@ <h3><a name="Watchdogs">4.4.5 Watchdog Timer Interfaces</a></h3>
   or <code>kill()</code> to communicate with NuttX tasks.
 </p>
 <ul>
-  <li><a href="#wdcreate">4.4.5.1 wd_create/wd_static</a></li>
-  <li><a href="#wddelete">4.4.5.2 wd_delete</a></li>
   <li><a href="#wdstart">4.4.5.3 wd_start</a></li>
```

Comment:
Paragraph number is wrong.  All need to be re-numbered starting at 1.

Reply:
Will fix in the next update.

Reply:
I will not merge without the fix.

Reply:
Done.